includes/class-sensei-utils.php

```diff
@@ -2626,6 +2626,34 @@ public static function format_last_activity_date( string $date ) {
 
 		return wp_date( get_option( 'date_format' ), $date->getTimestamp(), $timezone );
 	}
+
+	/**
+	 * Gets the HTML content from the Featured Video for a post.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param string $post_id the post ID.
+	 *
+	 * @return string The featured video HTML output.
+	 */
+	public static function get_featured_video_html( $post_id ) {
```

Comment:
Merged this method with #5701, so merging the `feature/learning-mode` branch should bring it in here.